src/ws-header/storage/local-storage.js

```diff
@@ -13,7 +13,7 @@ export class LocalStorage extends AbstractStorage {
    */
   set(key, value) {
     const encodedValue = encodeURIComponent(JSON.stringify(value));
-    localStorage.setItem(`${this.name}${key}`, encodedValue);
+    window.localStorage.setItem(`${this.name}${key}`, encodedValue);
```

Comment:
Why do you add this obsolete window object here?

Reply:
see above